src/Background/bookmarks.ts

```diff
@@ -0,0 +1,26 @@
+import * as api from '../utils/api/likerland';
+
+let bookmarks = new Set();
+
+export async function refreshBookmark() {
+  const newBookmarks = await api.refreshBookmarkList();
+  bookmarks = new Set(
+    newBookmarks.map((url: string) => {
```

Comment:
I think there is no need for type definition here as `api.refreshBookmarkList` provides